### PR TITLE
Editor Review of Chapter 11

### PR DESCRIPTION
**Pull request by amandamuledy**
Hi Oren! Erik has some suggestions in his commit. Here's my feedback:

Line 59--I think the meaning of these ellipses won't be 100% clear to all your readers. If you mean, "the list of things you'll have to do goes on and on," I would just say that. 

Line 520 ("In our Northwind example")--I think you need to specify what you're referring to. I have no memory of this reference, and if I don't, some others probably won't either. If you're going back to a past chapter, you should probably hyperlink.

General--I left "map-reduce" (which I was changing to "MapReduce") alone in places where I thought you might be referring readers to something specific, where having an exact match to what you show/use in RavenDB would be more helpful. (E.g., "In the `Indexes` tab, go to `Map-Reduce Visualizer`") But my advice is to change any reader-facing items you can to MapReduce to be consistent.

### CODE REVIEW

File: Ch11/Ch11.md [hunk 1]
```diff
@@ -1,156 +1,152 @@
...
 
 ### Executing simple aggregations
 
 Listing 11.1 shows a simple aggregation query, giving us the total number of orders and items purchased by a particular company.
 
 ```{caption="A simple aggregation query in RavenDB" .sql}
 from Orders as o
-group by o.Company
-where o.Company  = 'companies/1-A'
+group by o.company
```
Comment (ayende) [inline Ch11/Ch11.md:25]: Don't change casing of code elements.

**Review by ayende — COMMENTED** [review]
Line 59 - agreed, changed that.

Except for the map-reduce visualizer, it changed it all

**Comment by ayende** [comment]
My changes on top of yours 
https://github.com/ravendb/book/commit/8b2301bff18697a0236d18cf47261798f47d1da1
https://github.com/ravendb/book/commit/9c864dc9add0fc9fc6a61e9a202e42f2136b385c

**Comment by amandamuledy** [comment]
Oh, Oren, I'm so terribly sorry about the "company" thing. The only way I
can trigger spellcheck to engage in the markdown editor I'm using is to run
a find and replace (sigh, technology), and I happened to choose use that
word. I didn't realize it would mess with your code and classes.

**Pull request merged**